src/Text/Pandoc/Writers/Markdown/Inline.hs

```diff
@@ -557,6 +558,11 @@ inlineToMarkdown opts lnk@(Link attr@(ident,classes,kvs) txt (src, tit)) = do
       | useAuto -> return $ literal srcSuffix
       | otherwise -> return linktext
     _ | useAuto -> return $ "<" <> literal srcSuffix <> ">"
+      | useAuto && useWikilink -> return $ "[[" <> literal srcSuffix <> "]]"
```

Comment:
is there a reason for using this rather than the more portable <...> syntax?

Reply:
The motivation here was to preserve as much information as possible, esp the fact that this is a wikilink. That "wiki-autolink" syntax appears to enjoy wide support across implementations. But just falling back to explicit title/url wiki syntax could be saver, if a little less concise.